### ISSUE TITLE
Create more beautufil design for work exp

### ISSUE THREAD

**Issue by abhishekprakash256** [abhishekprakash256/frontend-component-kit#50]
## Stylize the work exp 

- work exp stylize 
- stack stylize

**Comment by abhishekprakash256** [comment]
like icons for development stacks

**Comment by abhishekprakash256** [comment]
import { SiMongodb, SiTypescript, SiJavascript, SiNginx } from 'react-icons/si';

this can be used for icons

**Comment by abhishekprakash256** [comment]
// File: TechStackIcons.jsx
import React from 'react';
import {
  SiPython, SiJavascript, SiTypescript, SiGnubash,
  SiReact, SiNextdotjs, SiNodedotjs, SiFlask, SiBootstrap, SiNginx,
  SiAmazonaws, SiDocker, SiGithubactions, SiGitlab, SiAnsible,
  SiPostgresql, SiMongodb, SiRedis, SiSqlite, SiMysql,
  SiPytorch, SiTensorflow, SiPandas, SiScikitlearn
} from 'react-icons/si';

import { FaBrain } from 'react-icons/fa';
import { GiArtificialIntelligence } from 'react-icons/gi';

const techStack = [
  // Programming Languages
  { name: 'Python', icon: <SiPython color="#3776AB" /> },
  { name: 'JavaScript', icon: <SiJavascript color="#F7DF1E" /> },
  { name: 'TypeScript', icon: <SiTypescript color="#3178C6" /> },
  { name: 'Shell', icon: <SiGnubash color="#89E051" /> },

  // Web Development
  { name: 'React', icon: <SiReact color="#61DAFB" /> },
  { name: 'Next.js', icon: <SiNextdotjs /> },
  { name: 'Node.js', icon: <SiNodedotjs color="#339933" /> },
  { name: 'Flask', icon: <SiFlask /> },
  { name: 'Bootstrap', icon: <SiBootstrap color="#7952B3" /> },
  { name: 'NGINX', icon: <SiNginx color="#009639" /> },

  // Cloud & DevOps
  { name: 'AWS', icon: <SiAmazonaws /> },
  { name: 'Docker', icon: <SiDocker color="#2496ED" /> },
  { name: 'GitHub Actions', icon: <SiGithubactions /> },
  { name: 'GitLab CI/CD', icon: <SiGitlab color="#FC6D26" /> },
  { name: 'Ansible', icon: <SiAnsible /> },

  // Databases
  { name: 'PostgreSQL', icon: <SiPostgresql color="#336791" /> },
  { name: 'MongoDB', icon: <SiMongodb color="#47A248" /> },
  { name: 'Redis', icon: <SiRedis color="#DC382D" /> },
  { name: 'SQLite', icon: <SiSqlite color="#003B57" /> },
  { name: 'SQL', icon: <SiMysql color="#4479A1" /> },

  // Machine Learning & AI
  { name: 'PyTorch', icon: <SiPytorch color="#EE4C2C" /> },
  { name: 'TensorFlow', icon: <SiTensorflow color="#FF6F00" /> },
  { name: 'Pandas', icon: <SiPandas color="#150458" /> },
  { name: 'Scikit-learn', icon: <SiScikitlearn color="#F7931E" /> },
  { name: 'Deep Learning', icon: <FaBrain /> },
  { name: 'Reinforcement Learning', icon: <GiArtificialIntelligence /> },
];

const TechStackIcons = () => {
  return (
    <div style={{ padding: '2rem' }}>
      <h2 style={{ textAlign: 'center', marginBottom: '1rem' }}>💻 My Tech Stack</h2>
      <div style={{
        display: 'grid',
        gridTemplateColumns: 'repeat(auto-fill, minmax(100px, 1fr))',
        gap: '1.5rem',
        justifyItems: 'center',
        alignItems: 'center',
        fontSize: '2rem'
      }}>
        {techStack.map((tech, index) => (
          <div key={index} style={{ textAlign: 'center' }}>
            {tech.icon}
            <div style={{ fontSize: '0.8rem', marginTop: '0.3rem' }}>{tech.name}</div>
          </div>
        ))}
      </div>
    </div>
  );
};

export default TechStackIcons;
